### PR TITLE
[BugFix][Variables]check null before use sessiioinVar in VariableMgr::dump()

### PR DESCRIPTION
**Pull request by fzhedu**
Signed-off-by: fzhedu <fzhedu@gmail.com>

## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #https://github.com/StarRocks/starrocks/issues/8296

## Problem Summary(Required) ：

`sessionVar` is used directly but maybe null, so check it first to avoid NPE.

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java [hunk 1]
```diff
@@ -495,7 +495,7 @@ public static List<List<String>> dump(SetType type, SessionVariable sessionVar,
 
                 // For session variables, the flag is VariableMgr.SESSION | VariableMgr.INVISIBLE
                 // For global variables, the flag is VariableMgr.GLOBAL | VariableMgr.INVISIBLE
-                if ((ctx.getFlag() > VariableMgr.INVISIBLE) && !sessionVar.isEnableShowAllVariables()) {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java:498]: better to provide comment when sessionVar could be null.

Comment (fzhedu) [reply fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java:498]: done, set the note front the function.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 8 / 8 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/VariableMgr.java|8|8|100.00%|

**Comment by Seaven** [comment]
run all

**Review by Seaven — APPROVED** [review]

**Comment by fzhedu** [comment]
run starrocks_fe_unittest

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**